### PR TITLE
Add Italian list of words.

### PR DESCRIPTION
**Pull request by ajdavidl**
I added a Italian list of words with the lists from the [wikwik](https://it.wikwik.org/tutteleparole.htm) and [listediparole](https://www.listediparole.it/tutteleparole.htm) sites.

I also changed src/WordLists.jl and test/runtests.jl to support the Italian language.

Finally, I updated the Readme file to include Italian.

### CODE REVIEW

File: src/WordLists.jl [hunk 1]
```diff
@@ -101,7 +102,11 @@ const LOOKUP_TABLE = Dict(
     "français" => "fr",
     "fre" => "fr",
     "fra" => "fr",
-    "fr" => "fr"
+    "fr" => "fr",
+    "italian" => "it",
+    "italiano" => "it",
```
Comment (LilithHafner) [inline src/WordLists.jl:107]: ~I'm not sure if "italien", "italiano", or both are correct.~

~I do not know the Italian language, but found a source supporting each of these~
~"J’apprends l’italien. I’m learning Italian." from [collinsdictionary](https://www.collinsdictionary.com/dictionary/french-english/italien)~
~"italiano m ([uncountable](https://en.wiktionary.org/wiki/Appendix:Glossary#uncountable)) the [Italian](https://en.wiktionary.org/wiki/Italian) [language](https://en.wiktionary.org/wiki/language)" from [wictionary](https://en.wiktionary.org/wiki/italiano#Noun_4)~

My mistake. I was looking at https://www.loc.gov/standards/iso639-2/php/code_list.php which lists the _french_ name for Italian, not the Italian name.

**Comment by LilithHafner** [comment]
Do you think it is appropriate to include conjugations like "scristianissi" in the word list?

**Comment by ajdavidl** [comment]
Yes, I think it is appropriate to include verb conjugations but I don´t know where we can find those words.

**Review by LilithHafner — APPROVED** [review]
Okay, great. Most conjugations are in the list already. For conjugations of Scristianire listed by https://www.italian-verbs.com/italian-verbs/conjugation.php?parola=scristianissi, I only found 2 out of 50 to be missing, "scristianirebbero" and "scristianirò". That is IMO a high enough quality to be better than nothing.

**Pull request merged**